### PR TITLE
Create a new service: OG role handler.

### PR DESCRIPTION
**Pull request by RoySegall**
#107

### CODE REVIEW

File: src/OgRolesHandler.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ * Contains /Drupal/og/OgRolesHandler
+ */
+
+namespace Drupal\og;
+
+use Drupal\Core\Session\AccountInterface;
+use Drupal\og\Entity\OgRole;
+use Drupal\og\Entity\OgUsersRole;
+
+class OgRolesHandler {
```
Comment (dawehner) [inline src/OgRolesHandler.php]: That classname sounds quite generic, any idea for a better name, maybe OgRolesStorage?

File: src/OgRolesHandler.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+   *   The account instance.
+   * @param $role
+   *   The role instance.
+   *
+   * @return OgUsersRole
+   */
+  public function grantRole(AccountInterface $account, OgRole $role) {
+    $users_role = OgUsersRole::create();
+    $users_role
+      ->setUid($account)
```
Comment (amitaibu) [inline src/OgRolesHandler.php]: maybe `setAccount` and `setRole`?

File: src/OgRolesHandler.php (hunk 1 above)
Comment (amitaibu) [inline src/OgRolesHandler.php]: also, maybe better to have an interface

**Comment by RoySegall** [comment]
@damiankloip I talked with @amitaibu about this one and we came to a cross road and we thought you could help us decide. What Iv'e done so far is a new content entity which keep the relation between the user and the OG role. 

But @amitaibu suggested another thing: when the og membership bind a user to the group, we can have a field which will keep all the roles the user have for that group.

**Pull request closed without merging**